test/functional/test_runner.py

```diff
@@ -41,11 +41,16 @@
     CROSS = "x "
     CIRCLE = "o "
 
-if os.name == 'posix':
+if os.name != 'nt' or sys.getwindowsversion() >= (10, 0, 14393):
+    if os.name == 'nt':
+        import ctypes
+        kernel32 = ctypes.windll.kernel32
+        kernel32.SetConsoleMode(kernel32.GetStdHandle(-11), 7)
```

Comment:
if you have to use ctypes directly here, with magic numbers, can you please at least add a comment about what the numbers here mean, what it is doing